packages/frontend/src/view/ApiDefs/Form/CacheTransforms.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { CacheTransforms as Props } from '../../../types';
+import { Col, FormGroup, Input, OutlineButton, Row } from '../../../ui';
+import { AddFieldArrayHeader } from '../../Form/AddFieldArrayHeader';
+import { useStyles } from './useStyles';
+import { CacheTransformsInvalidateOperations } from './CacheTransformsInvalidateOperations';
+
+export const CacheTransforms: React.FC<Props> = ({
+  cache,
+  onAddCache,
+  onRemoveCache,
+  register,
+  errors,
+  control,
+}) => {
+  const { formRow } = useStyles();
+
+  return (
+    <FormGroup title="Cache Transforms">
+      <AddFieldArrayHeader title="Cache Transform" onAddClick={onAddCache} />
+      {cache.map((cache: any, idx: number) => {
+        return (
+          <FormGroup key={`cacheTransform-${idx}`} title="Cache Transform">
+            <Row spacing={2} className={formRow}>
+              <Col xs={6}>
+                <Input
+                  ref={register()}
+                  name={`cache[${idx}].field`}
+                  label="Field"
```

Comment:
`helperText="The Type and Field to apply cache too. You can use wildcards, for example Query.*"`